### ISSUE TITLE
Class 'Twilio\Twiml' not found

### ISSUE THREAD

**Issue by zotopteam** [aloha/laravel-twilio#174]
I know this problem has been mentioned, but the problem is still unresolved, still report this error in the laravel 5.8,  How can I fix it？

when use：

```
            Twilio::call($phone, function($message) use ($text, $lang) {
                $message->say($text, ['voice' => 'alice', 'language'=>$lang, 'loop'=>2]);
            });

```
show this error

```
Class 'Twilio\Twiml' not found 

……/vendor/aloha/twilio/src/Twilio.php(86): Aloha\\Twilio\\Twilio->twiml(Object(Closure))
```

**Comment by zotopteam** [comment]
Interim solution：Downgrading twilio/sdk (5.42.2 => 5.41.0)

**Comment by zotopteam** [comment]
In my composer.json

```
    "require": {
        "php": "^7.2",
        "aloha/twilio": "^4.0",
        "twilio/sdk":"5.41.0",
```

ADD this line and composer update, Downgrading twilio/sdk (5.42.2 => 5.41.0)

` "twilio/sdk":"5.41.0",`

**Comment by hannesvdvreken** [comment]
Fixed in 4.0.4. It excludes those versions of the twilio sdk.
